packages/lesswrong/components/tagging/TagPageButtonRow.tsx

```diff
@@ -97,32 +98,52 @@ const TagPageButtonRow = ({tag, editing, setEditing, className, classes}: {
       e.preventDefault();
     }
   }
-  
+
+  const showEdit = !editing
+  const noEditKarmaTooLow = currentUser && currentUser.karma < tagMinimumKarmaPermissions.edit
+  const noEditNotAuthor = currentUser && !currentUser.isAdmin && tag.authorOnly && tag.userId !== currentUser._id
+  const canEdit = showEdit && !noEditKarmaTooLow && !noEditNotAuthor
+
   const editTooltip = <>
+    {noEditNotAuthor && <>
+      <div>
+      This article can only be edited by the author or an admin, please comment in the discussion to suggest changes
+    </div>
+    <br />
+    </>}
+    {noEditKarmaTooLow && <>
+      <div>
+      You must have at least {tagMinimumKarmaPermissions.edit} karma to edit this topic
+    </div>
+    <br />
+    </>}
     {!!numFlags && <>
       <div>
         This article has the following flag{tag.tagFlagsIds?.length > 1 ? "s" : ""}:{' '}
-        {tag.tagFlags.map((flag, i) => <span key={flag._id}>{flag.name}{(i+1) < tag.tagFlags?.length && ", "}</span>)}
+        {tag.tagFlags.map((flag, i) => <span key={flag._id}>{flag.name}{(i + 1) < tag.tagFlags?.length && ", "}</span>)}
       </div>
-      <br/>
+      <br />
     </>}
     <ContentItemBody
       className={classes.beginnersGuide}
-      dangerouslySetInnerHTML={{__html: beginnersGuideContentTag?.description?.html || ""}}
+      dangerouslySetInnerHTML={{ __html: beginnersGuideContentTag?.description?.html || "" }}
       description={`tag ${tag?.name}`}
     />
   </>
-  
+
   return <div className={classNames(classes.buttonsRow, className)}>
-    {!editing && (!currentUser || currentUser.karma >= tagMinimumKarmaPermissions.edit) && <LWTooltip
+    {showEdit && <LWTooltip
       className={classes.buttonTooltip}
       title={editTooltip}
     >
-      <a className={classes.button} onClick={handleEditClick}>
+      {canEdit ? (<a className={classes.button} onClick={handleEditClick}>
         <EditOutlinedIcon /><span className={classes.buttonLabel}>
           Edit
         </span>
-      </a>
+      </a>) : (
+        <a className={classes.disabledButton} onClick={() => {}}><LockIcon /><span className={classes.buttonLabel}>
```

Comment:
The solidness of the lock icon makes it appear a deeper grey than the other icons or text. You can lighten it till it optically looks right.

Reply:
This is the new lighter version:
<img width="441" alt="Screenshot 2022-08-09 at 16 27 54" src="https://user-images.githubusercontent.com/77623106/183702072-960907fc-6873-43a9-b9dc-d54f39426445.png">